Makefile

```diff
@@ -0,0 +1,12 @@
+
+# include .env file and export its env vars (-include to ignore error if it does not exist)
+-include .env
+
+deploy:
```

Comment:
good stuff, this is for doing stuff locally, right?  And those ${VARIABLE} are env vars, right? 

I've only used makefiles for ci.  

So how do you run these commands from the command line?? 🤔

Reply:
Yup!

So essentially, i just run 
`make deploy`

Then when it generates the contract address, i update the value in the .env file, before running `make verify`

There is some bash stuff i could do to copy the value from terminal and update into .env directly, or simply just pass it on to make verify - but sadly I forgot how. 

Will let you know when I figure it out again :)